cmd/command__id/main.go

```diff
@@ -0,0 +1,65 @@
+package main
```

Comment:
The directory should be `cmd/command_id/`, I think there's an accidental double underscore.